### ISSUE TITLE
Disabling revita for specific applications? 

### ISSUE THREAD

**Issue by Localhorst86** [MERLev/reVita#14]
Is it possible to disable revita and its hot keys when in specific applications/titles? Inputs for DevilutionX for example overlap with some of the hot keys for revita.

**Comment by MERLev** [comment]
Yeah, i had exact same issue i think. Unfortunately, atm it's not possible, but should be definitely changed in the future - if I'll find a bit of free time for this.
